resources/views/admin/post/index.blade.php

```diff
@@ -70,4 +80,25 @@
     </section>
     <!-- /.content -->
   </div>
+<script>
+  $(document).on('click','table tr td a',function (event) {
+    event.preventDefault();
+    var url = $(this).attr('href');
+    var this_button = $(this);
+    $.ajax({
+      url: url,
+      type: 'PUT', 
+      headers: {'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')},
+      dataType: 'json',
+    })
+    .done(function(data) {
+      if(data.status == {{App\Post::ENABLE}}) {
```

Comment:
Chú ý space trong `{{  }}`.
Anh thấy `App\Post::ENABLE` dùng lại khá nhiều. chưa kể đây là gọi Model tử View. sai nguyên tắc MVC.
Em có thể tạo một thuộc tính trong model Post. để lưu status
```php
var status = [
    'enable' => 1,
    'disable' => 0
];
```

Ở Controller này lấy biến này ra rồi parse qua view. ở View chỉ cần dùng array này được truyền từ Controller thôi.

Reply:
mình dùng var nó báo lỗi convention "the var keyword must not be used to declare a property"  là sao anh? Em chuyển qua dùng public luôn rồi anh.